Assignment4/aplepie-d190040/Task08.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# **Task 08: Completing missing data**
+
+# In[1]:
+
+
+get_ipython().system('pip install rdflib')
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+
+# In[2]:
+
+
+from rdflib import Graph, Namespace, Literal, URIRef
+g1 = Graph()
+g2 = Graph()
+g1.parse(github_storage+"/rdf/data01.rdf", format="xml")
+g2.parse(github_storage+"/rdf/data02.rdf", format="xml")
+
+
+# Tarea: lista todos los elementos de la clase Person en el primer grafo (data01.rdf) y completa los campos (given name, family name y email) que puedan faltar con los datos del segundo grafo (data02.rdf). Puedes usar consultas SPARQL o iterar el grafo, o ambas cosas.
+
+# In[10]:
+
+
+ns = Namespace("http://data.org#")
+
+for s, p, o in g1.triples((None, None, ns.Person)):
+    print(s)
+
+
+# In[4]:
+
+
+g1 = g1 + g2
+
```

Comment:
Careful: here you have not just completed the requested field, but ALL the graph.